flatpak_builder_lint/staticfiles/exceptions.json

```diff
@@ -1147,6 +1147,10 @@
     "net.lutris.Lutris": {
         "finish-args-flatpak-spawn-access": "the app predates this linter rule"
     },
+    "net.retrodeck.retrodeck": {
+        "appid-uses-code-hosting-domain": "app-id predates this linter rule",
```

Comment:
net.retrodeck doesn't seem a code hosting domain?

Reply:
also don't lie. this doesn't predate a linter rule, because there is none against that app-id.

Reply:
this need to be resolved, it's not an option